src/tree/set.rs

```diff
@@ -568,43 +568,43 @@ impl<T, C> TreeSet<T, C> where C: Compare<T> {
 }
 
 /// A lazy forward iterator over a set.
-pub struct SetItems<'a, T:'a> {
-    iter: Entries<'a, T, ()>
+pub struct Iter<'a, T:'a> {
+    iter: tree_map::Iter<'a, T, ()>
 }
 
 /// A lazy backward iterator over a set.
-pub struct RevSetItems<'a, T:'a> {
-    iter: RevEntries<'a, T, ()>
+pub struct RevIter<'a, T:'a> {
```

Comment:
Yeah, it's not great. @aturon has some shadowy plans to make this specific usecase a bit more ergonomic (iterator newtypes).